flow/connectors/postgres/cdc.go

```diff
@@ -112,6 +112,7 @@ func GetChildToParentRelIDMap(ctx context.Context, conn *pgx.Conn) (map[uint32]u
 
 // PullRecords pulls records from the cdc stream
 func (p *PostgresCDCSource) PullRecords(ctx context.Context, req *model.PullRecordsRequest) error {
+	logger := logger.LoggerFromCtx(ctx)
```

Comment:
As mentioned in the description, `p.logger` also takes from logger.LoggerFromCtx during getconnector, so not sure why it doesn't print the flow name but this does